craco.config.js

```diff
@@ -1,5 +1,5 @@
 const WebpackBar = require("webpackbar");
-const CracoLessPlugin = require("craco-less");
+const CracoAntDesignPlugin = require("craco-antd");
```

Comment:
Changed from CracoLessPlugin to CracoAntDesignPlugin which also includes CracoLessPlugin.
Will need to run `yarn install`.